code/parachain/frame/staking-rewards/src/test/prelude.rs

```diff
@@ -1,14 +1,17 @@
+use std::sync::Once;
+
 pub use crate::prelude::*;
 
 use composable_tests_helpers::test::block::MILLISECS_PER_BLOCK;
-use log::LevelFilter;
 pub use sp_core::{
 	sr25519::{Public, Signature},
 	H256,
 };
 
 #[cfg(test)]
 pub use composable_tests_helpers::test::currency::*;
+use tracing::metadata::LevelFilter;
```

Comment:
Feels over-engineered for local debugging of tests. Using the debugger should be better for unit tests.

Reply:
I personally don't like using debuggers with rust, and not all editors support running debuggers either. This is a universal solution that works for everyone

Reply:
I think personal opinions should not affect the code style for everyone. I use the debugger and it works